### DISCUSSION TITLE
Assenze docenti/ata

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#351] [Idee e proposte]
Buongiorno,ricordo che LAMPSCHOOL permetteva un'area di gestione delle assenze di tutto il personale docente e ata con relative ferie che potevano essere concesse o rifiutate o sospese dal dirigente  etc...Sarebbe possibile avere lo stesso in Giuaschool?

**Comment by trinko** [comment]
Non è prevista la gestione amministrativa nel registro.
